confidant/public/modules/resources/views/credential-details.html

```diff
@@ -146,6 +146,10 @@
         <button type="button" ng-disabled="editableForm.$waiting" ng-click="addTag()" class="btn pull-right">Add Tag</button>
       </span>
     </div>
+    <div class="form-group" ng-show="credential.id">
+      <label for="credentialNextRotationDate">Credential Next Rotation Needed Date</label>
+      <span id="credentialNextRotationDate">{{ credential.next_rotation_date }} ({{ credential.daysTillRotation }} days)</span> <span id="credentialNextRotationDateTooltip" class="glyphicon glyphicon-info-sign" data-toggle="tooltip" title="This date is determined by the last rotation date and the last decryption date."></span>
```

Comment:
we ok with showing a negative number here for expired creds?

Reply:
yep. I think it's good context that it should have been rotated a while ago.